point_test.go

```diff
@@ -160,8 +172,8 @@ var _ = Describe("Point", func() {
 
 		for i := 0; i < trials; i++ {
 			a, b = RandomPoint(), RandomPoint()
-			axf, ayf = a.XY()
-			bxf, byf = b.XY()
+			axf, ayf, _ = a.XY()
+			bxf, byf, _ = b.XY()
```

Comment:
Consider explicitly confirming that there is no error.

Reply:
Fixed in bf8a281.